docs/configuration.md

```
@@ -0,0 +1,600 @@
+## XP Block Drops
+
+```json
+{
+  "values": {
+    "xpDroppingBlock.<block_name>.minExp": 1,
+    "xpDroppingBlock.<block_name>.maxExp": 5
+  }
+}
+```
+
+## Snowball Damage
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)